env/scratch/kustomization.yaml

```diff
@@ -1,20 +1,26 @@
 bases:
-  - ../../base
-  - ../../base-karpenter
-
+  - ../../base/karpenter
+  - ../../base/kube-state-metrics
+  #- ../../base/prometheus-cloudwatch
```

Comment:
Do we still want these commented out or should we straight up delete the lines? I guess it's useful to have the commented out lines for when we want to include and tests this code in scratch?

Reply:
Yeah I have them commented just so I can easily add them back when I need to test those specific functions